### DISCUSSION TITLE
🍄XSOSI 计划 3 月份数据

### DISCUSSION THREAD

**Discussion by bifenglin** [X-lab2017/open-wonderland#267] [2️⃣ 聊项目]
3月份数据如下：
![image](https://user-images.githubusercontent.com/15813364/229711032-21c775ec-a6de-4fc7-bfa8-86f6d76b8af5.png)   

![image](https://user-images.githubusercontent.com/15813364/229711050-c311e14f-aa04-4ea2-a93b-08ae252048ac.png)
  
整体上涨趋势很明显

**Comment by frank-zsy** [comment]
<img width="1306" alt="image" src="https://user-images.githubusercontent.com/8512426/230396965-d56ee51b-7029-4872-9a9a-438436948dc0.png">

我的问题，在之前计算项目内 OpenOpenRank 时由于本月引入了评论点赞中的 P2P 的价值转移，但因为代码中的 bug，事实上计算并没有在 40 次迭代中收敛，修正 bug 后的数据如上所示。

**Comment by zhicheng-ning** [comment]
已同步更新 XSOSI 大屏：https://dataease.nzcer.cn/link/E1OQKVBz

![image](https://user-images.githubusercontent.com/39022409/230399803-ca25bdb0-a5f6-42b8-8e34-1af2dff15f3d.png)

**Comment by frank-zsy** [comment]
<img width="1376" alt="image" src="https://user-images.githubusercontent.com/8512426/230414538-facaa0e0-683a-4c6d-bfe9-c76fbe3d535f.png">

再做一个极小的微调
